provision/acc_provision/templates/aci-containers.yaml

```diff
@@ -680,6 +680,65 @@ spec:
                     minimum: 1
                     maximum: 1023
 ---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
General comment: There shoudl be only one CR that does enable/disable of droplogs. If this is not possible with CRD and web-hook, configmap may be a better option

Reply:
As discussed, we will have a CRD with node selector and selectively enable droplog. We keep track of all CRs defined and then merge. If any CR has the feature disabled, then it’s disabled irrespective of other CRs.